src/views/Shipment.vue

```diff
@@ -179,16 +182,17 @@ export default defineComponent({
         }
       })
     },
+    updateProductCount(payload: any){
+      this.store.dispatch('shipment/updateShipmentProductCount',{payload})
```

Comment:
Improve code formatting.

Reply:
Done Sir.
Thank you sir for reviewing my code.